src/Blueprint/Blueprint/ChangeLog.md

```diff
@@ -0,0 +1,21 @@
+<!--
+    Please leave this section at the top of the change log.
```

Comment:
>Please [](start = 4, length = 6)

Should we fill this out? What do other modules do with these files?

Reply:
Added brief change log text

---
In reply to: [257091072](https://github.com/Azure/azure-powershell/pull/8537#discussion_r257091072) [](ancestors = 257091072)